site/package.json

```diff
@@ -6,9 +6,9 @@
     "build": "prettier --check . && next build",
     "start": "next start",
     "lint": "next lint",
-    "lingui:extract": "cmd=\"NODE_ENV=test npx lingui extract --clean --overwrite 2>&1 | grep 'Cannot process file'\"; if [ -z \"$(eval $cmd)\" ]; then exit 0; else exit 1; fi",
-    "lingui:compile": "NODE_ENV=test lingui compile",
-    "lingui:extract:dev": "NODE_ENV=test npx lingui extract --clean --overwrite --locale=en",
+    "lingui:extract": "node ../scripts/lingui.js extract --clean --overwrite",
+    "lingui:compile": "node ../scripts/lingui.js compile",
```

Comment:
Fair enough :)

Reply:
I removed the `NODE_ENV=test` before the `lingui compile` command because it works on my PC. I doi not remember why I put it in the first place. Let me know if it causes issues with CI or something.

Reply:
@biwano https://github.com/KlimaDAO/klimadao/pull/43#discussion_r771928052